build.sh

```
@@ -0,0 +1,69 @@
+cp icon64.png "$EXTENSION_DIR/"
+cp jszip.min.js "$EXTENSION_DIR/"
+cp LICENSE "$EXTENSION_DIR/"
+
+# 複製目錄
+cp -r bolt "$EXTENSION_DIR/"
+cp -r claude "$EXTENSION_DIR/"
+cp -r gemini "$EXTENSION_DIR/"
+cp -r lovable "$EXTENSION_DIR/"
+
+# 創建 ZIP 文件
+cd "$TEMP_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)